judo/controller/controller.py

```diff
@@ -241,6 +262,25 @@ def update_traces(self) -> None:
             elites[sensor :: self.num_trace_sensors] = s1
         self.traces = np.reshape(elites, (total_traces_rollouts, 2, 3))
 
+    def _init_action_normalizer(self) -> Normalizer:
+        """Initialize the action normalizer."""
+        action_normalizer_kwargs = {}
+        if self.action_normalizer_type == "min_max":
+            action_normalizer_kwargs["min"] = self.task.actuator_ctrlrange[:, 0]
+            action_normalizer_kwargs["max"] = self.task.actuator_ctrlrange[:, 1]
+        elif self.action_normalizer_type == "running_mean_std":
+            action_normalizer_kwargs["init_std"] = 1.0
+        return make_normalizer(self.action_normalizer_type, self.model.nu, **action_normalizer_kwargs)
+
+    def update_action_normalizer(self, actions: np.ndarray, rewards: np.ndarray) -> None:
+        """Update the action normalizer."""
+        if normalizer_registry.get(self.action_normalizer_type) != type(self.action_normalizer):
+            # Force the normalizer to be re-initialized when the type changes in GUI
+            self.action_normalizer = self._init_action_normalizer()
```

Comment:
two comments:
1. the logic for checking on the normalizer should probably happen directly in the control loop instead of being attached to this function, which combines two functionalities on the normalizer together
2. only checking for the type is potentially not robust in a future case where we have parameterized normalizers and want to adjust them live in the gui. but, it's not important at all right now, so you should just leave a todo comment explaining that we should potentially check for changes in a normalizer config and update when appropriate (it will not always be appropriate, because if, for example, you only change `init_std`, then you don't want the entire normalizer to reset, you only want the next time you reset the normalizer to use different args to the constructor)